All project code/index.js

```diff
@@ -335,10 +335,37 @@ async function cityToCoordinates(locationInput) {
     // Perform city/country conversion to latitude/longitude coordinates here.
     const country = locationInput.country;
     const city = locationInput.city;
+    var latitude = 0;
+    var longitude = 0;
+    // axios call to API-Ninja geocoding api to return latitude and longitude from city and country input 
+    await axios({
+        // url for API-Ninja
+        url:'https://api.api-ninjas.com/v1/geocoding?city=',
```

Comment:
Why does your url have ?city= at the end? I think this should be handled in axios in the params field, unless that's some really specific requirement in the api.

Reply:
I had just copied this from the API-Ninja Documentation, but it works without "?city=" so I will remove it.